build/Targets/Settings.props

```diff
@@ -45,7 +45,6 @@
     <VisualStudioReferenceAssemblyVersion Condition="'$(VisualStudioReferenceAssemblyVersion)' == ''">$(VisualStudioReferenceMajorVersion).0.0.0</VisualStudioReferenceAssemblyVersion>
     <VisualStudioCodename>Dev$(VisualStudioReferenceMajorVersion)</VisualStudioCodename>
     <MinimumVisualStudioVersion>$(VisualStudioVersion)</MinimumVisualStudioVersion>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
See the convo in infra swat.  It's a tension issue between making F5 simple and build conistency on developer machines.